docs/proposals/proposal-doc-template.md

```diff
@@ -0,0 +1,29 @@
+
+
+## Overview 
+
+> 1 – 2 sentences of what this document will outline.
+
+## Problem Statement & Motivation 
+
+> State the problem or challenge in a way that ties back to the target user. What are the goals and non-goals? Why does this matter to them? Make the problem real & relevant. We can link to an existing GitHub issue for clarification or justification.
+
+## Background & Context 
```

Comment:
According to the description, I feel that the `Background and Context` section overlaps somewhat with the `Problem Statement and Motivation` section. For example, to write the problem and motivation, the background, context, and users will be described. Can you clarify the difference between the two?

Reply:
I elaborated more in this section: 
Describes the broader landscape, including prior solutions and constraints.